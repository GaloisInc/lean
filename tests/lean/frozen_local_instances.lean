example (α : Type) [has_add α] [has_mul α] [decidable_eq α] : true :=
begin
  (tactic.frozen_local_instances >>= tactic.trace),
  tactic.unfreeze_local_instances,
  (tactic.frozen_local_instances >>= tactic.trace),
  trivial
end
