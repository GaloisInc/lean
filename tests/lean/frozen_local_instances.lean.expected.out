(some [_inst_3, _inst_2, _inst_1])
none
